pkg/storaged/stratis-details.jsx

```diff
@@ -207,6 +216,153 @@ export const StratisPoolDetails = ({ client, pool }) => {
+                                        .then(std_reply);
+                            });
+                }
+            }
+        });
+    }
+
+    function change_passphrase() {
+        with_keydesc(client, pool, (keydesc, keydesc_set) => {
+            dialog_open({
+                Title: _("Change passphrase"),
+                Fields: [
+                    PassInput("old_passphrase", _("Old passphrase"),
+                              {
+                                  visible: vals => !keydesc_set,
+                                  validate: val => !val.length && _("Passphrase cannot be empty")
+                              }),
+                    Skip("medskip"),
+                    PassInput("new_passphrase", _("New passphrase"),
+                              { validate: val => !val.length && _("Passphrase cannot be empty") }),
+                    PassInput("new_passphrase2", _("Confirm"),
+                              { validate: (val, vals) => vals.new_passphrase.length && vals.new_passphrase != val && _("Passphrases do not match") })
+                ],
+                Action: {
+                    Title: _("Save"),
+                    action: vals => {
+                        function rebind() {
+                            return get_unused_keydesc(client, pool.Name)
+                                    .then(new_keydesc => {
+                                        return with_stored_passphrase(client, new_keydesc, vals.new_passphrase,
+                                                                      () => pool.RebindKeyring(new_keydesc))
+                                                .then(std_reply);
+                                    });
+                        }
+
+                        if (vals.old_passphrase) {
+                            return with_stored_passphrase(client, keydesc, vals.old_passphrase, rebind);
+                        } else {
+                            return rebind();
+                        }
+                    }
+                }
+            });
+        });
+    }
+
+    function remove_passphrase() {
+        dialog_open({
+            Title: _("Remove passphrase?"),
+            Body: <div>
+                <p className="slot-warning">{ fmt_to_fragments(_("Passphrase removal may prevent unlocking $0."), <b>{pool.Name}</b>) }</p>
+            </div>,
+            Action: {
+                DangerButton: true,
+                Title: _("Remove"),
+                action: function (vals) {
+                    return pool.UnbindKeyring().then(std_reply);
+                }
+            }
+        });
+    }
+
+    function add_tang() {
+        return with_keydesc(client, pool, (keydesc, keydesc_set) => {
+            dialog_open({
+                Title: _("Add Tang keyserver"),
+                Fields: [
+                    TextInput("tang_url", _("Keyserver address"),
+                              {
+                                  validate: validate_url
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230601-113309-0e7aeb29-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)